build.gradle.kts

```diff
@@ -31,8 +32,18 @@ allOpen {
     annotations("javax.persistence.Entity", "javax.persistence.MappedSuperclass", "javax.persistence.Embedabble")
 }
 
+val codeArtifactToken = System.getenv("CODEARTIFACT_PAT")
+    ?: "aws codeartifact get-authorization-token --domain erop-artifacts --domain-owner 063998039290 --query authorizationToken --output text --profile code-artifact".runCommand()
```

Comment:
Simplify if possible, and possibly can pass the profile as environment param instead of the `CODEARTIFACT_PAT`